prov/net/src/xnet_progress.c

```diff
@@ -783,6 +783,37 @@ static void xnet_run_ep(struct xnet_ep *ep, bool pin, bool pout, bool perr)
 	};
 }
 
+static void xnet_progress_cqe(struct xnet_uring *uring,
+			      ofi_io_uring_cqe_t *cqe)
+{
+	struct ofi_bsock *bsock;
+
+	assert(xnet_io_uring);
+	bsock = (struct ofi_bsock *) cqe->user_data;
+	assert(bsock);
+}
```

Comment:
I'm hoping a future patch expands this function to do more, yes?

Reply:
Yes, this is correct. I will be adding the code that completes and progresses TX/RX operations in a future PR.